### PR TITLE
Add warning message when changing coe.adaptor

### PR DESCRIPTION
**Pull request by timdengyun**
Add a warning message when changing coe.adaptor to invalid value.

### CODE REVIEW

**Review by jwsui — DISMISSED** [review]
LGTM

**Review by salv-orlando — COMMENTED** [review]
LGTM, a minor comment inline

File: pkg/controller/configmap/config.go [hunk 1]
```diff
@@ -70,6 +70,10 @@ func (adaptor *ConfigMapK8s) FillDefaults(configmap *corev1.ConfigMap, spec *con
 		log.Error(err, "failed to load ConfigMap")
 		return err
 	}
+
+	if cfg.Section("coe").HasKey("adaptor") && cfg.Section("coe").Key("adaptor").Value() != "kubernetes" {
+		log.Info(fmt.Sprintf("coe.adaptor is not supported to change the value other than kubernetes, ignored this config option"))
```
[the remark this answers is not in the document]
Comment (timdengyun) [reply pkg/controller/configmap/config.go]: Thanks for advice.

File: pkg/controller/configmap/config.go [hunk 2]
```diff
@@ -70,6 +70,10 @@ func (adaptor *ConfigMapK8s) FillDefaults(configmap *corev1.ConfigMap, spec *con
 		log.Error(err, "failed to load ConfigMap")
 		return err
 	}
+
+	if cfg.Section("coe").HasKey("adaptor") && cfg.Section("coe").Key("adaptor").Value() != "kubernetes" {
+		log.Info(fmt.Sprintf("The operator infers the adaptor to use from the environment. Any setting for coe.adaptor will be ignored"))
```
Comment (heypnus) [inline pkg/controller/configmap/config.go:75]: `fmt.Sprintf()` is not needed.

Comment (timdengyun) [reply pkg/controller/configmap/config.go]: Thanks, fixed.

**Review by heypnus — APPROVED** [review]

**Pull request merged**